entities/communication/voiceCallSetting.json

```diff
@@ -0,0 +1,99 @@
+{
+    "type": "object",
+    "properties": {
+        "uid": {
+            "type": "string",
+            "description": "The voice call settings unique identifier."
+        },
+        "staff_uid": {
+            "type": "string",
+            "description": "Receiving forwarded call staff uid."
+        },
+        "forward_number": {
+            "type": "string",
+            "description": "The phone number of the staff receiving the forwarded call."
+        },
+        "dedicated_number": {
+            "type": "string",
+            "description": "The dedicated number of the business receiving calls from clients."
+        },
+        "forwarding_enabled": {
+            "type": "boolean",
+            "description": "Is forwarding the voice call enabled or not."
+        },
+        "call_forwarding_policy": {
+            "type": "enum",
+            "values": [
+                "ALWAYS",
+                "WORK_HOURS",
+                "NEVER"],
+            "description": "When to forward the call based on staff availability."
+        },
+        "staff_weekly_availability_uid": {
+            "type": "string",
+            "description": "Staff weekly availability details uid."
+        },
+        "call_timeout_sec": {
+            "type": "number",
+            "description": "Call ringing timeout in seconds."
+        },
+        "voice_scripts": {
+            "type": "json",
+            "description": "Voice scripts to play for caller."
+        },
+        "external_app_config": {
+            "type": "json",
+            "description": "External AI app details."
+        },
```

Comment:
A configuration JSON object defining the external AI app managing the voice call. The object defines three keys:
 app_name: the name of the app used (for internal use)
 full_url: the endpoint related to that app
 method: the HTTP method that should be used (GET/POST)